api/models.py

```diff
@@ -1,3 +1,13 @@
 from django.db import models
 
-# Create your models here.
+
+class Todo(models.Model):
+    author = models.ForeignKey('auth.User', related_name='todo', on_delete=models.CASCADE)
+    created_date = models.DateTimeField(auto_now_add=True)
+    state = models.CharField(max_length=100, blank=False)
```

Comment:
Use choices field from Django and set following options:
- Todo
- In Progress
- In Review
- Done

https://docs.djangoproject.com/en/4.1/ref/models/fields/#choices